app/assets/javascripts/views/return_to_draft_view.js

```diff
@@ -12,36 +7,49 @@ ELMO.Views.ReturnToDraftView = class ReturnToDraftView extends ELMO.Views.Applic
       'click .return-to-draft-link': 'handleLinkClicked',
       'shown.bs.modal #return-to-draft-modal': 'handleModalShown',
       'click #return-to-draft-modal .btn-primary': 'handleAcceptClicked',
+      'keypress #override': 'handleKeypress',
```

Comment:
The diff for this file is dense. See the individual commit for clarity: https://github.com/thecartercenter/nemo/pull/720/commits/ed6a8dd8cf11e1eaeed7e82d0a73d6e747afb3fa